### ISSUE TITLE
gulp serve:dist not working

### ISSUE THREAD

**Issue by nutanlade** [angular-fullstack/generator-angular-fullstack#2435]
- [x] I understand that GitHub issues are not for tech support, but for questions specific to this generator, bug reports, and feature requests.

Item | Version
----- | -----
generator-angular-fullstack | 3.7.6
Node | 5.4.1
npm | 3.10.4
Operating System | OS X 10
etc | etc

Item | Answer
----- | -----
Transpiler | Babel
Markup | HTML
CSS | CSS
Router | ngRoute 
Client Tests | Jasmine
DB | MongoDB
Auth | Y
etc | etc

My application is working fine with gulp serve but when I'm running through gulp serve:dist facing below issue. Please help me to resolve the issue.

Error: [$injector:unpr] Unknown provider: authInterceptorProvider <- authInterceptor <- $http <- $templateFactory <- $view <- $state

**Comment by Awk34** [comment]
Which generator-angular-fullstack version did you scaffold with? You can find it in your `.yo-rc.json` as `"generatorVersion"`.

**Comment by nutanlade** [comment]
generatorVersion is 3.7.6

**Comment by nutanlade** [comment]
My index.html file looks like below.

`<script src="app/admin/admin.module.js"></script>
    <script src="components/auth/auth.module.js"></script>
    <script src="components/util/util.module.js"></script>
    <script src="app/account/account.js"></script>
    <script src="app/account/login/login.controller.js"></script>
    <script src="app/account/serviceprovidersignup/serviceprovidersignup.controller.js"></script>
    <script src="app/account/settings/settings.controller.js"></script>
    <script src="app/account/signup/signup.controller.js"></script>
    <script src="app/admin/admin.controller.js"></script>
    <script src="app/admin/admin.router.js"></script>
    <script src="app/app.constant.js"></script>
    <script src="app/billservice/billservice.controller.js"></script>
    <script src="app/billservice/billservice.js"></script>
    <script src="app/directives.js"></script>
    <script src="app/employeedashboard/employeedashboard.controller.js"></script>
    <script src="app/employeedashboard/employeedashboard.js"></script>
    <script src="app/employeedashboard/securitycomplaints/securitycomplaints.controller.js"></script>
    <script src="app/employeedashboard/securitycomplaints/securitycomplaints.js"></script>
    <script src="app/employeedashboard/visitors/visitors.controller.js"></script>
    <script src="app/employeedashboard/visitors/visitors.js"></script>
    <script src="app/main/main.controller.js"></script>
    <script src="app/main/main.js"></script>
    <script src="app/profile/profile.controller.js"></script>
    <script src="app/profile/profile.js"></script>
    <script src="app/superadmin/superadmin.controller.js"></script>
    <script src="app/superadmin/superadmin.js"></script>
    <script src="components/auth/auth.service.js"></script>
    <script src="components/auth/interceptor.service.js"></script>
    <script src="components/auth/router.decorator.js"></script>
    <script src="components/auth/user.service.js"></script>
    <script src="components/footer/footer.controller.js"></script>
    <script src="components/footer/footer.directive.js"></script>
    <script src="components/modal/modal.service.js"></script>
    <script src="components/mongoose-error/mongoose-error.directive.js"></script>
    <script src="components/navbar/navbar.controller.js"></script>
    <script src="components/navbar/navbar.directive.js"></script>
    <script src="components/oauth-buttons/oauth-buttons.controller.js"></script>
    <script src="components/oauth-buttons/oauth-buttons.directive.js"></script>
    <script src="components/socket/socket.service.js"></script>
    <script src="components/util/util.service.js"></script>`

**Comment by Awk34** [comment]
In `dist` mode your `index.html` should only have 2 script tags IIRC. There should be an app bundle and a vendor bundle. Try searching around the closed GitHub issues for similar questions, I believe people have had the same problem before. Be sure to ensure you only use one kind of line ending on all of your source files, as well.

**Comment by nutanlade** [comment]
@Awk34 above code is from gulp serve. In dist file I have only 2 script tags (app and vendor). I have gone through various closed Github issues and tried to apply on my code. I didn't worked. I think after app.js all components*.js should come. For that I modified sort function still no use. Now after gulp build I modified app.js such that "interceptor.service.js" code is after app.js, then javascript error is resolved. But when starting application getting 404 errors for all pages. 90% of my development is almost completed and now I'm facing this issue.
